src/plugins/intel_gpu/src/plugin/compiled_model.cpp

```diff
@@ -354,12 +354,16 @@ bool CompiledModel::isSerializable() {
     return true;
 }
 
+// Cache blob format:
+//     [ ConstInputsDataMap / ConstOutputsDataMap ]
+//     [ ov::Node::Input/ ov::Node::Output ]
+//     [ ov::intel_gpu::Graph ]
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017908598
I added cache blob descriptions here, Graph, network, primitive_inst, typed_primitive_impl_ocl and typed_primitive_onednn_impl. Thank you.